src/snowflake/snowpark/udtf.py

```diff
@@ -533,9 +533,7 @@ def register_from_file(
         self,
         file_path: str,
         handler_name: str,
-        output_schema: Union[
-            StructType, Iterable[str], Tuple[PandasDataFrameType, Iterable[str]]
-        ],
+        output_schema: Union[StructType, Iterable[str], "PandasDataFrameType"],
```

Comment:
Do we no longer support `Tuple[PandasDataFrameType, Iterable[str]`, or is this part of the type hint fix?

Reply:
Right, the previous one is wrong, this is part of the type hint fix.